src/settings/accounts-settings.tsx

```diff
@@ -284,16 +235,6 @@ export const AccountsSettings = (): React.JSX.Element => {
 
 		let modifyPrefsRequest: ModifyPrefsRequest | undefined;
```

Comment:
Now this is not used anymore. Consider also to clean the type AccountSettingsBatchRequest/Response, which is specific for this setting and can be cleaned.